### DISCUSSION TITLE
`~/.ideavimrc` doesn't seem to work with IntelliJ Client when using JetBrains Gateway

### DISCUSSION THREAD

**Discussion by quebin31** [JetBrains/ideavim#980] [Q&A]
IdeaVim plugin version: 2.16.0 (Installed on Client)
IntelliJ version: 2024.2.0.1 (Client)
Gateway version: 2024.2

I've already tried putting my `.ideavimrc` in both the home directory of my client and the host, but none of these seem to be working, I'm using the following file:

```
set relativenumber
set number
set ideajoin

set clipboard+=unnamed
set clipboard+=ideaput
```

I can only install the plugin in the client, installing it in the host makes the editor interaction janky and unstable (cursor jumping and not acting correctly)

**Comment by citizenmatt** [comment]
The plugin must be installed in the client only. If you install it in the host, there will be issues. As far as I'm aware, `ideavimrc` should work ok for the client - it should be in the home directory on the client machine, not on the host. Can you provide some more details about what's not working from the file when installed on the client?

And please note that IdeaVim does have a number of known issues with Gateway - there are several problems that can affect typing and other features. We're trying to work on that now.

**Comment by quebin31** [comment]
Thanks for the response, yes, I have it installed in the client only, and my client can find my `.ideavimrc` file, but it looks like it's not being sourced automatically, it only works after I run `:source ~/.ideavimrc` and only for the currently opened editor. 

I'm also aware of some of the issues IdeaVim has with Gateway, previously it was almost impossible to use it when installed in the client because of some cursor problems, now it seems to be working way better, the only thing I noticed that doesn't seem to be fully working is the error I'm reporting here, thanks!

**Comment by eraustud** [comment]
Pinging to say that this is still an issue. I can't disable the error bell in IdeaVim because the ~/.ideavimrc is not utilized by the client, which is an infuriating experience.

My plugin is also installed on the client only.
